### DISCUSSION TITLE
Initialise Pixi vue/vite plugin?

### DISCUSSION THREAD

**Discussion by spotd** [quasarframework/quasar#16347] [General - Components / Directives / etc]
How can i initialise the brilliant Vue 3 pixi framework as per instructions [here ->](https://vue3-pixi.vercel.app/guide/introduction.html#initialize-vue-plugin)

Is a great vue 3 integration but this would usually sit in the vite.config.js. I cant figure out how to translate correctly to work with quasar.config?

**Comment by TobyMosque** [comment]
https://quasar.dev/quasar-cli-vite/handling-vite/#adding-vite-plugins

**Comment by spotd** [comment]
I have tried with this method, without success so far

**Reply by TobyMosque** [reply]
Probably, u'll need to extract the vue plugin and extend that.

**Comment by spotd** [comment]
Good shout I’ll try that

Mark Dodson
Creative Director
mobile: + 44 7702573283
email: ***@***.***
web: www.spotd.uk

**Reply by MarkOnDuty** [reply]
Did you get anywhere with this? I've been trying to do something similar with a [vite-imagetools](https://github.com/JonasKruckenberg/imagetools/blob/main/docs/guide/getting-started.md), which is even simpler, and have hit a wall.

**Reply by spotd** [reply]
Hi Yes, I meant to put my solution here (eventually)

So i did get it working via quasar.config  ->

` vitePlugins: [
         [import('./src/boot/pixi.mjs')]
      ]`

_pixi.mjs_

```
import { defineConfig } from 'vite';
import { compilerOptions, transformAssetUrls } from 'vue3-pixi';

export default async ({ app, router, store, vue }) => {
  defineConfig({
    plugins: [vue({ template: { compilerOptions, transformAssetUrls } })],
  })
}
```

But i was still getting some webGL render errors, so decided to just implement pixi js raw so its completely transparent. That way i have all errors cleared. 

Probably no help to you
